.github/workflows/presubmit.yaml

```diff
@@ -102,5 +102,24 @@ jobs:
       with:
         name: testlogs
         path: e2etests/bazel-testlogs
+  build-x86_64-docker-image:
```

Comment:
Move the step to https://github.com/google/android-cuttlefish/blob/main/.github/workflows/artifacts.yaml which run on post-submits. Follow the name pattern there:  `export-docker-image-x86_64`.

Reply:
Done.